.github/workflows/ci-tests.yml

```diff
@@ -349,3 +349,102 @@ jobs:
             - name: Clean up database (reverse-only import)
               run: nominatim refresh --postcodes --word-tokens
               working-directory: /home/nominatim/nominatim-project
+
+    install-no-superuser:
+      runs-on: ubuntu-latest
+      needs: create-archive
+
+      strategy:
+          matrix:
+              name: [Ubuntu-22]
+              include:
+                  - name: Ubuntu-22
+                    image: "ubuntu:22.04"
+                    ubuntu: 22
+                    install_mode: install-apache
+
+      container:
+          image: ${{ matrix.image }}
+          env:
+              LANG: en_US.UTF-8
+
+      defaults:
+          run:
+              shell: sudo -Hu nominatim bash --noprofile --norc -eo pipefail {0}
+
+      steps:
+          - name: Prepare container (Ubuntu)
+            run: |
+                export APT_LISTCHANGES_FRONTEND=none
+                export DEBIAN_FRONTEND=noninteractive
+                apt-get update -qq
+                apt-get install -y git sudo wget
+                ln -snf /usr/share/zoneinfo/$CONTAINER_TIMEZONE /etc/localtime && echo $CONTAINER_TIMEZONE > /etc/timezone
+            shell: bash
+
+          - name: Setup database users
+            run: |
+                useradd -m nominatim
+                useradd -m osm-import
```

Comment:
You don't need this additional unix user. Simply run the no-superuser import as user 'nominatim' as well. Then the permission issues go away.

Reply:
Password authentication should work when you connect to 127.0.0.1 instead of the socket, .i.e. NOMINATIM_DATABASE_DSN="pgsql:dbname=nominatim;host=127.0.0.1;user=osm-import;password=osm-import"

Reply:
That was the hint I was looking for!
Thanks!